source/PokerSetup.py

```diff
@@ -0,0 +1,344 @@
+#   Code still needs to incorporate a pot.
+
+from source.CardGames import *
+import sys
+
+PlayersStillInGameList = PlayerList
+BlindsList = PlayersStillInGameList
+PlayersPlayingRoundList = PlayersStillInGameList
+PlayersOutOfGameList = []
+
+class PokerGameSetup:
+
+    def __init__(self, name, money: int = 0):
+        self.name = []
+        self.hand = []
+        self.money = money
+
+    def preGame(self, players: PlayerList):
```

Comment:
I think Andrew and I both built functions to cover most of this. We can check it out later. I don't see why either way would have any issues though :)